repository_cli/cli/click_options.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 Graz University of Technology.
+#
+# repository-cli is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Commonly used options for CLI commands."""
+
+import click
+
+option_identifier = click.option(
+    "--identifier",
+    "-i",
+    required=True,
+    help="metadata identifier as JSON",
+)
+
+option_pid = click.option(
+    "--pid",
+    "-p",
+    metavar="PID_VALUE",
+    required=True,
+    help="persistent identifier of the object to operate on",
+)
+
+# invenio repository records identifiers list -p "fcze8-4vx33"11:20
```

Comment:
That's a nice idea of adding the example commands: we can use them to document.
also add for the data an actual json example:

```suggestion
# invenio repository records identifiers list -p "fcze8-4vx33"
```
or does ```11:20``` mean something?